src/components/JobListing/JobListing.tsx

```diff
@@ -5,9 +5,10 @@ import { JobListingObj } from '../../types/JobListingObj';
 interface Props {
   jobsData: JobListingObj[];
   addFilter: (term: string) => void;
+  filters: string[];
 }
 
-export function JobListing({ jobsData, addFilter }: Props) {
+export function JobListing({ jobsData, addFilter, filters }: Props) {
```

Comment:
A bit of prop drilling starting from App...will look to replace this once useContext is setup